code/game/objects/structures/dropship_ammo.dm

```diff
@@ -413,3 +413,132 @@
+	icon_state = "m64"
+	ammo_id = "m64"
+	point_cost = 50
+
+/obj/structure/ship_ammo/cas_bomb/m64/detonate_on(turf/impact, attackdir = NORTH)
+	impact.ceiling_debris_check(3)
+	explosion(impact, 2, 3, 4, 6, small_animation = TRUE) //relatively weak
+	qdel(src)
+
+//Beg bomb
+/obj/structure/ship_ammo/cas_bomb/mark84
+	name = "\improper Mark 84 bomb"
+	desc = "The mark 84 or BLU-117 is a venerable specimen of military technology, being a rather large, unguided and cheap bomb. At least, for its size. It has seen many, many decades of service, especially in places where accuracy isn't needed."
+	icon_state = "mark84"
+	ammo_id = "MARK84"
+	point_cost = 175
+
+/obj/structure/ship_ammo/cas_bomb/mark84/detonate_on(turf/impact, attackdir = NORTH)
+	impact.ceiling_debris_check(3)
+	explosion(impact, 4, 6, 8, 10) //more spread out, with flames
+	qdel(src)
+
+//Mine laying bomb
+
+/obj/structure/ship_ammo/cas_bomb/minelaying
+	name = "\improper M43 'ML' bomb"
+	desc = "A small, blocky cylinder with a number of smaller tubes strapped to its sides. As the handsome and smart marine you are, you recognize this as a mine laying bomb, meant to dispere its payload over a wide area."
+	icon_state = "default_bomb"
+	ammo_id = "m43"
+	point_cost = 300
+
+/obj/structure/ship_ammo/cas_bomb/minelaying/detonate_on(turf/impact, attackdir = NORTH)
+
+	playsound(impact, 'sound/effects/explosionfar.ogg', 10, 1, 4)
+	impact.visible_message("<span class='notice'>You hear the far away crack of an explosion.</span>")
+
+	impact.ceiling_debris_check(2)
+	addtimer(CALLBACK(src, .proc/drop_mine, impact), 1.5 SECONDS)
+	if(!ammo_count)
+		QDEL_IN(src, travelling_time) //deleted after last minirocket has fired and impacted the ground.
+
+/obj/structure/ship_ammo/cas_bomb/minelaying/proc/drop_mine(turf/impact)
+	playsound(loc, 'sound/weapons/guns/interact/mortar_unpack.ogg', 25, 1)
+	var/list/impact_coords = list(list(-3,3),list(0,4),list(3,3),list(-4,0),list(4,0),list(-3,-3),list(0,-4), list(3,-3))
+	var/turf/T
+	var/list/coords
+	var/obj/item/explosive/mine/planted_mine
+	for(var/i=1 to 8)
+		coords = impact_coords[i]
+		T = locate(impact.x+coords[1],impact.y+coords[2],impact.z)
+		T.ceiling_debris_check(2)
+		new/obj/item/explosive/mine(T)
+		planted_mine.attack_self()
+	qdel(src)
+
+///
+///CLUSTER
+///
+
+/obj/structure/ship_ammo/cas_bomb/cluster
+	name = "\improper U402 'Cluster' bomb"
+	desc = "A cluster bomb which blows up before smothering a wide area with bomblets."
+	icon_state = "cluster"
+	ammo_id = "U402"
+	point_cost = 300
+	var/strikes_amount = 18 //the amount of bombs it drop. This
+
+/obj/structure/ship_ammo/cas_bomb/cluster/detonate_on(turf/impact, attackdir = NORTH)
+	impact.ceiling_debris_check(2)
+	addtimer(CALLBACK(src, .proc/cluster_impacts, impact), 1.5 SECONDS)
+
+/obj/structure/ship_ammo/cas_bomb/cluster/proc/cluster_impacts(turf/impact)
+	for(var/i = 1 to strikes_amount)
```

Comment:
why is strikes_amount not a define even?

Reply:
Not used to defines, mostly.

Reply:
Defines are apparently reserved for things which are checked in multiple places, so i guess i'll leave this as a variable